### PR TITLE
form page 2 (dropdown list version)

### PR DESCRIPTION
**Pull request by ymloh**
[Trello card](https://trello.com/c/UHAd6AMw/165-dev-sml-update-page-2-of-form-to-reflect-updated-wireframe)

Dropdown list (or Select) version of page 2 of the form.

### CODE REVIEW

**Comment by ymloh** [comment]
Sorry about closing and reopening! I realised I had branched off the page 3 branch rather than main and was trying to rebase it, but changed my mind since it's not likely page 3 will change. 😅

**Comment by hqtan** [comment]
Nice work on this version of Question 2 @ymloh ! :+1: 

I agree with @batteries76 : I prefer this version over the [initial one you spiked on](https://github.com/CodeforAustralia/council-emissions-calculator-spike/pull/14), (but I'd like to keep the combobox/autocomplete idea somewhere so we can revisit it!)

A few other things to note:

- If a respondent indicated in Question 1 that he/she only worked from home and didn't commute, the respondent should have been presented with a different view of Question 2 (one that says 'Looks like you are working from home'). see this [screenshot](https://trello.com/1/cards/611baad25119441c17d03320/attachments/6121e9fa9b10da5df83cd6f7/download/image.png) from the [page 3 update trello card](https://trello.com/c/73XNVrEa). This is my bad sorry! I forgot to add that detail in the [page 2 update work](https://trello.com/c/UHAd6AMw)

- the 'Continue' button should be disabled (preventing the respondent from proceeding to Question 3) if a valid travel method has not been selected

- likewise, the 'Continue' button in Question 3 should also be disabled if user hasn't provided a value for commute distance

I like what I'm seeing so far! :heart_eyes:

File: pages/form/Question3.js [hunk 1]
```diff
@@ -1,99 +1,46 @@
...
+            placeholder="Distance in kms"
+            value="25"
+            onChange={(e) => setKm(e.target.value)}
+          />
+        </NumberInput>
+        <FormHelperText id="km-input-helper">*Required</FormHelperText>
+      </FormControl>
+      <Grid templateColumns="repeat(2, 1fr)" gap={4}>
+        <BackButton href="/form/Question2" onClick={saveAnswers} />
+        <ContinueButton href="/form/Question4" onClick={saveAnswers} />
```
Comment (hqtan) [inline pages/form/Question3.js:42]: Hi @ymloh ,

Are you missing a `disabled` parameter in `<ContinueButton />` component?
The 'Continue' button is currently not disabled when the respondent hasn't provided a value in text field

Also, i'm not sure why, but I can't input a value in the text field!!
Can you try out the app with your changes here, and lemme know if you get to same problems?
https://council-emissions-calculator-spike-git-fork-605881-civic-makers.vercel.app/council-emissions-calculator-spike/

**Review by hqtan — CHANGES_REQUESTED** [review]

**Comment by hqtan** [comment]
Hi @ymloh ,
 
Another thing I noticed when hitting the 'Back' button on `form/Question3` is:
- (as expected) I get sent back to `form/Question2` :heavy_check_mark: 
- (not as expected) the transport mode I've previously supplied is no longer there :heavy_multiplication_x: 

Can you confirm the issue in [the preview app here](https://council-emissions-calculator-spike-git-fork-605881-civic-makers.vercel.app/council-emissions-calculator-spike/), and investigate? 

Thx! :bow:

File: pages/form/Question3.js (hunk 1 above)
Comment (ymloh) [reply pages/form/Question3.js:42]: Oh wow, that's pretty bad, sorry about that! My head must have been someplace else entirely.
I'm on it! ⛑️

Comment (ymloh) [reply pages/form/Question3.js]: @hqtan I used a Number Field for Question 3's input, so it won't accept non-numeric input. Is this the issue? Should I change it to string input?

**Comment by batteries76** [comment]
This is all great work you two. I'll keep my beak out of this unless I'm needed to run a final eye over things. And if you are both happy and that's not necessary, then that's fantastic. Sorry to be hands off, but that feels like the right play with things going well between you two and my slightly limited time today.

File: pages/form/Question3.js (hunk 1 above)
Comment (hqtan) [reply pages/form/Question3.js]: oooh gotcha, sorry @ymloh ! I probably didn't notice that! :bow:

**Comment by hqtan** [comment]
Thx for the updates @ymloh !
I like it **even moar** now! :heart_eyes:

**Review by hqtan — APPROVED** [review]

**Pull request merged**